.storybook/main.js

```diff
@@ -5,6 +5,8 @@ const exportToCodesandboxAddon = require('storybook-addon-export-to-codesandbox'
 const { getAllPackageInfo, isConvergedPackage } = require('@fluentui/scripts/monorepo');
 const semver = require('semver');
 
+const { loadWorkspacePlugin } = require('../scripts/storybook');
```

Comment:
this is needed so TS understands the source without need to register it to `tsconfig.base.json`